cmd/context.go

```diff
@@ -7,7 +7,7 @@ import (
 	"syscall"
 )
 
-// ContextWithSignal create a context cancelled when SIGINT or SIGTERM are notified
```

Comment:
So we target 🇺🇸English instead of 🇬🇧 English right? (ref. https://www.grammarly.com/blog/canceled-vs-cancelled/ ). 

Not a blocking thing nor to be changed. Just wanted to check :)

Reply:
Yes exactly

Reply:
👍